screens/Misc.tsx

```diff
@@ -0,0 +1,86 @@
+import Checkbox from 'expo-checkbox';
+import React, { useState } from 'react';
+import { StyleSheet, Text, View, Pressable} from 'react-native';
+import CustomButton from '../components/customPostButton';
+
+
+export default function Misc({navigation}: {navigation: any})
```

Comment:
You can add more comments to let other cooperators know what the code is doing. Without any comments in a code file or function, in the future, you may also forget how these codes work.

Add comments to functions to let developers know what the function does. Ex:
```
/**
 * Adds two numbers.
 * @param {number} num1 The first number to add.
 * @param {number} num2 The second number to add.
 * @return {number} The result of adding num1 and num2.
 */
function bigThing() {
    // ...
}
```

Reply:
Yes, I will leave more comments

Reply:
Need to add more comments but left some for now, will work on locations and add comments as I work